tools/memleak.py

```diff
@@ -522,17 +530,33 @@ def print_outstanding():
                 if args.show_allocs:
                         print("\taddr = %x size = %s" %
                               (address.value, info.size))
-        to_show = sorted(alloc_info.values(),
-                         key=lambda a: a.size)[-top_stacks:]
+        to_show = None
+        if sort_key == "size":
+                to_show = sorted(alloc_info.values(),
+                                 key=lambda a: a.size)[-top_stacks:]
+        elif sort_key == "count":
+                to_show = sorted(alloc_info.values(),
+                                 key=lambda a: a.count)[-top_stacks:]
+        else:
+                print("Fatal: reached unreachable code");
+                exit()
```

Comment:
to_show = sorted(alloc_info.values(), key = sort_keys[sort_key]) [-top_stacks:]

Reply:
Awesome! The code becomes one line. Thanks!
I applied your suggestion.